scientific/src/commonMain/kotlin/unit/DimensionlessUnit.kt

```diff
@@ -0,0 +1,109 @@
+ * or a quantity of dimension one,
+ * with a corresponding constant of measurement in the SI of the constant one (or 1), which is not explicitly shown.
+ *
+ * "
+ * 2.3.3 Dimensions of quantities
+ * ...
+ * There are quantities Q for which the defining equation is such that all of the dimensional
+ * exponents in the equation for the dimension of Q are zero.
+ * This is true in particular for any quantity that is defined as the ratio of two quantities of the same kind.
+ * For example, the refractive index is the ratio of two speeds and the relative permittivity is
+ * the ratio of the permittivity of a dielectric medium to that of free space.
+ * Such quantities are simply numbers. The associated constant is the constant one, symbol 1,
+ * although this is rarely explicitly written (see 5.4.7).
+ * ...
+ * The constant one is the neutral element of any system of units – necessary and present automatically.
+ * There is no requirement to introduce it formally by decision.
+ * Therefore, a formal traceability to the SI can be established through appropriate,
+ * validated measurement procedures.
+ *
+ * 5.4.7 Stating quantity values being pure numbers
+ *
+ * As discussed in Section 2.3.3, values of quantities with constant one, are expressed simply as numbers.
+ * The constant symbol 1 or constant name “one” are not explicitly shown.
+ * SI prefix symbols can neither be attached to the symbol 1 nor to the name “one”,
+ * therefore powers of 10 are used to express particularly large or small values.
+ *
+ * Quantities that are ratios of quantities of the same kind (for example length ratios and amount fractions)
+ * have the option of being expressed with units (m/m, mol/mol) to aid the understanding of
+ * the quantity being expressed and also allow the use of SI prefixes, if this is desirable (μm/m, nmol/mol).
+ * Quantities relating to counting do not have this option, they are just numbers.
+ *
+ * The internationally recognized symbol % (percent) may be used with the SI.
+ * When it is used, a space separates the number and the symbol %.
+ * The symbol % should be used rather than the name “percent”.
+ * In written text, however, the symbol % generally takes the meaning of “parts per hundred”.
+ * Phrases such as “percentage by mass”, “percentage by volume”, or “percentage by amount of substance” shall not be used;
+ * the extra information on the quantity should instead be conveyed in the description and symbol for the quantity.
+ * "
+ * Source:  "SI Brochure: The International System of Units, 9th Edition".
+ */
+@Serializable
+
+sealed class Dimensionless : ScientificUnit<PhysicalQuantity.Dimensionless>,
+    MetricAndImperialScientificUnit<PhysicalQuantity.Dimensionless>
+
+val DimensionlessUnits: Set<Dimensionless> get() = setOf(
+    One,
+    Percent,
+    Permill
+)
+
+object One : Dimensionless(), MetricBaseUnit<MeasurementSystem.MetricAndImperial, PhysicalQuantity.Dimensionless> {
```

Comment:
What if I need two? Also the real value below is from `value`, maybe better to be `Countable`?

Reply:
"One" is a name of the Unit (from the official SI docs) not the value, please read the comment above

Reply:
I see, but I can do 2(One) so two items of One unit? If should be One it should be strictly value = 1, and I should convert to any other value and init it with another value than 1

Reply:
*shouldn't

Reply:
Countable unit, 123(Countable) vs 123(One)

Reply:
Counting is an example of quantity not a Unit:
>There are also some quantities that cannot be described in terms of the seven base quantities of the SI, but have the nature of a count. 
> ...
> Counting quantities are also quantities with the associated unit one.

Please have a look at https://www.bipm.org/en/publications/si-brochure/  (those are the people that literally maintain SI standard).

In short, what you are asking for is factually incorrect. 
I have an impression that you're confusing:
- Quantity, where we could have a Countable Quantity if needed 
  (as a specialisation / subset of Dimensionless quantity)
  [for more see: https://en.wikipedia.org/wiki/Dimensionless_quantity, https://en.wikipedia.org/wiki/Physical_quantity ]
- Unit (of Measure) in this case Unit of "One" 
  (There was a proposal to name the unit "Uno" instead of "One" https://link.springer.com/article/10.1007/s00769-010-0747-9 but since it didn't happen "One" is still the official term for the base unit of dimensionless quantities.
- Magnitude which is numerical the value in statement like `1 kg`

I know that this is complex but at the end of the day Metrology is an independent field of Science for a reason.

Reply:
TL;DR: The Unit with name "One" is an official SI unit. 
I may not like it, you may not like it, others may not like it but it is what it is.

Reply:
Then it shouldn't have any value, and have symbol `1`, right?

Reply:
What value have a Celsius unit?
When it comes to the symbol `1` you are correct that `1` is a symbol of unit "One", but
SI documentation says:
![image](https://user-images.githubusercontent.com/4947911/150986617-7a7ab2e3-4db8-4a72-ac7c-fe34b3a09116.png)

Since, afaik, `val symbol: String` is always used when displaying scientific values I left it empty.
This should change when we have formatters for scientific units as this would allow us to properly define symbol for an unit like "One" but at the same time be in the accordance with the standard and not to display `1`

Reply:
Thanks for details explaining

Reply:
I think it is useful to remember that, by necessity, we mix here different "realms", a mathematical, physical and linguistic one. Where we have:
- Observable phenomena and units used in measurement
- Their corresponding mathematical objects (quantities, numbers)
- The names and symbols used for the linguistic description of both

Imo, unit "One" is a corner case because we had to use same word to refer to different concepts